floor_generator/lib/processor/entity_processor.dart

```diff
@@ -266,6 +266,8 @@ class EntityProcessor extends Processor<Entity> {
       return '$parameterValue as String';
     } else if (parameterType.isDartCoreInt) {
       return '$parameterValue as int';
+    } else if (parameterType.getDisplayString() == 'Uint8List') {
```

Comment:
💡Please make an extension function out of this.

```dart
extension Uint8ListTypeChecker on DartType {
  @nonNull
  bool get isUint8List => getDisplayString() == 'Unit8List';
}

// usage
parameterType.isUint8List;
```

Reply:
Done!